.eslintrc.js

```diff
@@ -244,15 +244,15 @@ module.exports = {
               {
                 target: [
                   '(src|x-pack)/plugins/**/*',
-                  '!(src|x-pack)/plugins/*/server/**/*',
```

Comment:
modified to add support for:
x-pack/legacy/plugins/**reporting/export_types/csv**/server/